ui/package.json

```
@@ -2,6 +2,7 @@
     "name": "tk-2022-ui",
     "version": "0.0.1",
     "scripts": {
+        "start": "npm run dev --open",
```

`npm run dev --open` doesn't work
```suggestion
        "start": "svelte-kit dev --open",
```